child_compassion/models/project_lifecycle_event.py

```diff
@@ -112,7 +112,7 @@ def create(self, vals):
         if event.type == 'Reactivation':
             if fund_suspended:
                 project.reactivate_project()
-            if hold_gifts and not event.hold_gifts():
+            if hold_gifts and not event.hold_gifts:
```

Comment:
It won't always be False, `event.hold_gifts` refers to its value in the incoming message while `hold_gifts` refers its current value. The test prevents the method from being called if the incoming message doesn't change the `hold_gift`'s value. I believe the parentheses were a typo.